app/components/application/AppEndpointCard/AppEndpointCard.tsx

```diff
@@ -1,26 +1,29 @@
+import { Flex, Grid } from "@mantine/core"
```

Comment:
We should be pulling from pocket-blocks everywhere we can. I see Flex isnt included in our pocket-blocks tho. Based on how you are using Flex, you could just handle that with a Box and marginBottom, or a Stack component